score-v1b1.json

```
@@ -0,0 +1,340 @@
+          }
+        },
+        "httpProbe": {
+          "description": "An HTTP probe details.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "path"
+          ],
+          "properties": {
+            "host": {
+              "description": "Host name to connect to. Defaults to the container IP.",
+              "type": "string"
+            },
+            "scheme": {
+              "description": "Scheme to use for connecting to the host (HTTP or HTTPS). Defaults to HTTP.",
+              "type": "string",
+              "enum": [
+                "HTTP",
+                "HTTPS"
+              ]
+            },
+            "path": {
+              "description": "The path of the HTTP probe endpoint.",
+              "type": "string"
+            },
+            "port": {
+              "description": "The path of the HTTP probe endpoint.",
+              "type": "number"
+            },
+            "httpHeaders": {
+              "description": "Additional HTTP headers to send with the request",
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "name": {
+                    "description": "The HTTP header name.",
+                    "type": "string"
+                  },
+                  "value": {
+                    "description": "The HTTP header value.",
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "resources": {
+      "description": "The dependencies needed by the Workload.",
+      "type": "object",
+      "minProperties": 1,
+      "additionalProperties": true,
+      "patternProperties": {
+        "^*": {
+          "description": "The resource name.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": [
+            "type"
+          ],
+          "properties": {
+            "type": {
+              "description": "The resource in the target environment.",
+              "type": "string"
+            },
+            "metadata": {
+              "description": "The metadata for the resource.",
+              "type": "object",
+              "minProperties": 1,
+              "additionalProperties": true,
+              "properties": {
+                "annotations": {
+                  "description": "Annotattions that apply to the property.",
+                  "type": "object",
+                  "minProperties": 1,
+                  "additionalProperties": {
+                    "type": "string"
+                  }
+                }
+              }
+            },
+            "properties": {
+              "description": "DEPRECATED: The properties that can be referenced in other places in the Score Specification file.",
```

```suggestion
              "description": "DEPRECATED: The properties that can be referenced in other places within the Score Specification file.",
```
do you mean to say `within the Score` file? Im not very much into Score yet 😬 so I don't completely understand this sentence, but probably is my lack of knowledge.

---

This was copied from our official documentation. Can't advise here )